### ISSUE TITLE
Hello World fails.

### ISSUE THREAD

**Issue by rdebath** [AYIDouble/Brainfuck-Interpreter-Node.js#2]
This program should print "Hello World!" not random control characters.
``` bf
+[>[<-[]>+[>+++>[+++++++++++>][>]-[<]>-]]++++++++++<]>>>>>>----.<<+++.<-..+++.<-.>>>.<<.+++.------.>-.<<+.<.
```

**Comment by IDouble** [comment]
I tested "Hello World!" with both of these Tools and it's working.

https://copy.sh/brainfuck/text.html

```
-[------->+<]>-.-[->+++++<]>++.+++++++..+++.[--->+<]>-----.---[->+++<]>.-[--->+<]>---.+++.------.--------.-[--->+<]>.
```

https://www.dcode.fr/brainfuck-language

```
++++++++++[>+>+++>+++++++>++++++++++<<<<-]>>>++.>+.+++++++..+++.<<++.>+++++++++++++++.>.+++.------.--------.<<+.
```

![Brainfuck Interpreter Hello World](https://github.com/AYIDouble/Brainfuck-Interpreter-Node.js/blob/master/Images/Brainfuck-Interpreter-Node.js-Hello-World.png)

What is the official Charset of Brainfuck? ANSI-8? or UTF-16?

**Comment by rdebath** [comment]
Yes, well, those "Hello World!" programs just contain simple loops, my one is much more annoying 😁 

As I wrote [on the Esolangs Wiki](https://esolangs.org/wiki/Brainfuck#Character_set) it is a little obscure what character set should be used. Occasionally, you get programs that output binaries (for example a [BF compiler that outputs Linux ELF executables](https://github.com/matslina/awib)) but usually the output characters are plain ASCII, so any 8-bit character set works. The few Russians, Chinese and Japanese I've seen seem to use UTF-8. Nobody uses UTF-16. All variations use just a character 10 as the newline character.
